diesel/src/pg/types/date_and_time/std_time.rs

```diff
@@ -53,7 +53,7 @@ fn usecs_to_duration(usecs_passed: u64) -> Duration {
 
 fn duration_to_usecs(duration: Duration) -> u64 {
     let seconds = duration.as_secs() * USEC_PER_SEC;
-    let subseconds = duration.subsec_nanos() / NANO_PER_USEC;
+    let subseconds = duration.subsec_micros();
```

Comment:
This seems unrelated, but :+1: if it doesn't bump our stable version

Reply:
Clippy complained about it ;)